posthog/hogql/errors.py

```diff
@@ -58,7 +58,7 @@ class QueryError(ExposedHogQLError):
     pass
 
 
-class NotImplementedError(InternalHogQLError):
+class NotImplementedHogQLError(InternalHogQLError):
```

Comment:
If these get "HogQL" in the name, then it probably makes sense for "ParsingError" to also be "HogQLParsingError" for the same clarity